lesson21/tailwind.config.cjs

```diff
@@ -0,0 +1,19 @@
+module.exports = {
+  content: [
+    './src/**/*.{html,js}'
+  ],
+  theme: {
+    fontFamily:{
+      body:[
+        "Helvetica Neue",
+        "Arial",
+        "Hiragino Kaku Gothic ProN",
+        "Hiragino Sans",
+        "Meiryo",
+        "sans-serif"
+      ]
```

Comment:
I learned how to specify fonts! Amazing!!!! :)

Reply:
I agree with Sae. Thanks! It was helpful.